README.md

```diff
@@ -50,11 +50,10 @@ Planned features:
     (divide [this x y] (/ x y)))
 
 ;; A proxied implementation of MathService for use via RPC
-(defservice remote-math-service
+(def-remote-service remote-math-service
```

Comment:
I feel like it's pretty unusual to have hyphens in a `def...` macro. I'm not sure it's much of a concern, but I thought I'd bring it up.